### ISSUE TITLE
Toast Alerts SR reads "close" button

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5449]
Toast Alerts (and actually all alerts with buttons) are reading the button immediately after the message, not much semantic sense, so the close button should be out of the role="alert/status"

<img width="821" alt="image" src="https://user-images.githubusercontent.com/7963804/229103170-4a45f726-9361-4524-8154-89a86aeae303.png">

**Comment by matuzalemsteles** [comment]
Hmm maybe we have to change the `role="alert"` to just be inside the text. Maybe it's a little difficult but I think it's possible because there are some buttons that are inside the text to do the alignment but I think we can change without problems.
